iris-dni/iris-frontend
# Feature/362 fallback og image

## Story

Add Fallback for `og:image`, when no image is present. Additionally a Bug in the [`src/server.js`](https://github.com/iris-dni/iris-frontend/pull/275/files#diff-e6a5b42b2f7a26c840607370aed5301a) was fixed too (`HTTP_AUTH_ENABLE` handling).

## How To Test

- add an image to the `ogFallbackImage` property in the [`src/settings/config.js`](https://github.com/iris-dni/iris-frontend/pull/275/files#diff-3fdd34795a1c2b4e5365e92182030794) file
- open a test location and check out the `og:image` meta-tag
